vpr/src/route/overuse_report.cpp

```diff
@@ -4,6 +4,11 @@
 #include "globals.h"
```

Comment:
Add a file level comment about what this file is for, and what kinds of routines should go in it.